marimo/_cli/export/commands.py

```diff
@@ -101,6 +102,16 @@ async def start() -> None:
     type=bool,
     help="Include notebook code in the exported HTML file.",
 )
+@click.option(
```

Comment:
if you aren't using this yet, maybe we can remove it until i add a warning when opening it.
(you can keep `run_app_then_export_as_reactive_html` still if you want)

Reply:
sounds good, will revert.